src/Contexts/TodoContext.tsx

```diff
@@ -1,6 +1,11 @@
-import React, { createContext, useState } from "react";
+import React, { createContext, useState, useEffect } from "react";
 import { ReactNode } from "react";
-import { Task } from "../App";
+
+interface Task {
+  id: number;
+  name: string;
+  isComplete: boolean;
+}

```

Comment:
Remover, pois utilizávamos na importando antigamente

```suggestion
import { Task } from "../App";

```